src/components/CurrencyInputPanel/index.tsx

```diff
@@ -179,7 +179,7 @@ export default function CurrencyInputPanel({
                     style={{ display: 'inline' }}
                   >
                     {!hideBalance && !!currency && selectedCurrencyBalance
-                      ? 'Balance: ' + selectedCurrencyBalance?.toSignificant(6)
+                      ? `${t('Balance')}: ${selectedCurrencyBalance?.toSignificant(6)}`
```

Comment:
can we use interpolation here

Reply:
I think both are okay, interpolation is easier to handle spaces.

Reply:
sorry, i mean in the translation as discussed above, e.g.  `t('Balance: {{balance}}', {balance: selectedCurrencyBalance?.toSignificant(6)})`

Reply:
Aha, good idea

Reply:
The code has been improved, but because the clone is used as i18n-next "char to split namespace from key", so I removed ":"

https://github.com/i18next/react-i18next/issues/985

Reply:
This seems to be another reason to use arbitrary keys instead of the english translation (which includes the colon). I would strongly prefer that. Would you be able to make that change?

Reply:
These separator options are configurable. 
I still prefer to use English directly as the key to making everything simple.

Reply:
In fact, there is no problem with using arbitrary keys. It mainly depends on core developers to maintain i18n keys so that the community can participate in translation easily. But now it is very difficult to participate in translation.

There are many Chinese Fork Uniswap projects appearing in the Chinese community because the official Uniswap i18n is difficult to translate directly.